src/action.js

```diff
@@ -24,6 +25,8 @@ const {
 
 module.exports = async function run() {
   try {
+    toolkit.logActionRefWarning('fastify/github-action-merge-dependabot')
```

Comment:
can we grab the argument from somewhere so that we don't hardcode it?

Reply:
I will check if that's possible

Reply:
I have updated the code and removed the hard-coded repo name. The toolkit automatically detects it.